Formula/tractorgen.rb

```diff
@@ -18,4 +18,16 @@ def install
     system "./configure", "--prefix=#{prefix}"
     system "make", "install"
   end
+
+  test do
+    expected = <<-'EOS'
```

Comment:
Please use `<<~'EOS'` in case we start auditing `<<-`.

Reply:
<<~'EOS' seems to strip whitespaces in the string when all lines start with whitespaces, as is the case here. Am I missing something?

Reply:
Hmm... I guess we'll just have to figure out how to deal with this one later if we do add such an audit.